packages/components/src/atoms/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes, PropsWithChildren } from 'react'
+
+interface Size {
+  width: string
+  height: string
+}
+
+export type BorderStyle = 'regular' | 'pill' | 'circle'
+
+export interface SkeletonProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+  /**
+   * Control whether skeleton should be visible or not.
+   */
+  loading?: boolean
+  /**
+   * Control whether the shimmer effect should be displayed or not.
+   */
+  shimmer?: boolean
+  /**
+   * Specifies the skeleton element size (width, height).
+   */
+  size: Size
+  /**
+   * Specifies the skeleton element border radius ('regular' | 'pill' | 'circle').
+   */
+  border?: BorderStyle
+  /**
+   * Customizes the skeleton element border size.
+   */
+  customBorder?: string
+}
+
+const Skeleton = forwardRef<HTMLDivElement, PropsWithChildren<SkeletonProps>>(
+  function Skeleton(
+    {
+      testId = 'fs-skeleton',
+      loading = true,
+      shimmer = false,
```

Comment:
Looking at the stater's PR I realized that shimmer as true is very common, so what do you think about set shimmer as true as default?

Reply:
nicee!